lua/lazyvim/plugins/extras/lang/python.lua

```diff
@@ -0,0 +1,116 @@
+return {
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if type(opts.ensure_installed) == "table" then
+        vim.list_extend(opts.ensure_installed, { "ninja", "python", "rst", "toml" })
```

Comment:
with rust, we should be avoiding duplicates (toml), not saying it's a problem for you to fix but it can be one for users down the road

Reply:
LazyVim already handle duplicated config.

But maybe makes sense extract toml to another plugin

Reply:
that's fine then..didn't know that actually

Reply:
yep, this is fine. It used to be an issue, but I fixed that in LazyVim